lib/Backend/Opnd.cpp

```diff
@@ -3180,11 +3207,14 @@ Opnd::Dump(IRDumpFlags flags, Func *func)
         IndirOpnd *indirOpnd = this->AsIndirOpnd();
 
         Output::Print(_u("["));
-        indirOpnd->GetBaseOpnd()->Dump(flags, func);
+        if (indirOpnd->GetBaseOpnd())
+        {
+            indirOpnd->GetBaseOpnd()->Dump(flags, func);
+            Output::Print(_u("+"));
```

Comment:
I thought about the placement of + being weird in some scenarios, but felt people can figure it out. However, as Tom suggests I can put null here, which seems reasonable, since otherwise we can't distinguish between index and base in case scale is 0. So I'll go with that, solution which should also solve issue of the +